@here/harp-datasource-protocol/lib/Expr.ts

```diff
@@ -696,10 +696,14 @@ export class ContainsExpr extends Expr {
 export class CallExpr extends Expr {
     descriptor?: OperatorDescriptor;
 
-    constructor(readonly op: string, readonly children: Expr[]) {
+    constructor(readonly op: string, readonly args: Expr[]) {
         super();
     }
 
+    get children() {
```

Comment:
In some cases, we reference `expr.children` in others `expr.args`.
Is there difference?, can't we settle on one one property name ?

Reply:
`children` is the generic name that we use for "AST nodes" containing a list of child nodes. In this case I introduced `args` to make sure that the patch was smaller and easier to read (e.g. instead of `args` the code now uses `call.args`); at some time I added a gettter `children` to keep the code backward compatible.